.devcontainer.json

```
@@ -0,0 +1,8 @@
+{
+    "name": "Dev Container Definition - AWS JSII",
+    "image": "jsii/superchain:1-buster-slim",
```

If I understand the documentation correctly, adding `remoteUser` might make `EPERM` problems go away (ref: https://code.visualstudio.com/remote/advancedcontainers/add-nonroot-user).

```suggestion
    "image": "jsii/superchain:1-buster-slim",
    "remoteUser: "superchain",
```